audit-ci.json

```diff
@@ -1,6 +1,5 @@
 {
     "low": true,
     "package-manager": "auto",
-    "registry": "https://registry.npmjs.org",
-    "allowlist": [1769]
```

Comment:
In order to do this you also need to update the `mssql` module. This vuln is not the axios one. Easiest to do as I mentioned, update all the modules in the `package.json` to the lest before regenerating.